packages/hardhat-core/src/internal/hardhat-network/provider/vm/dual.ts

```diff
@@ -190,6 +190,7 @@ export class DualModeAdapter implements VMAdapter {
     block: Block,
     config: RpcDebugTracingConfig
   ): Promise<RpcDebugTraceOutput> {
+    // TODO add rethnet adapter
```

Comment:
@Wodann I'm assuming we don't want dual mode tests due to the expected differences in output?

Reply:
@Wodann after our discussion I tried to make comparisons with `ethereumjs` output, but turns out the `ethereumjs` output doesn't match the fixtures (this is probably why a bunch of debug trace tests were disabled), so I gave up.

Reply:
@fvictorio is there a reason that ethereumjs output wouldn't match the fixtures? Is it post-processed?